src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,89 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final int DEFAULT_GROW = 2;
+    private static final float LOAD_FACTOR = 0.75f;
+
+    private Node<K, V>[] table;
+    private int size;
+    private int capacity;
+
+    public MyHashMap() {
+        table = ( Node<K, V>[]) new Node[DEFAULT_CAPACITY];
+        capacity = table.length;
```

Comment:
```suggestion
        capacity = DEFAULT_CAPACITY;
```
It will be more readable.